llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -852,12 +852,22 @@ void SIInstrInfo::copyPhysReg(MachineBasicBlock &MBB,
     }
 
     if (!AMDGPU::SReg_32RegClass.contains(SrcReg)) {
+      // When calling convention allocates SGPR for i1 argument, we may
+      // have a SRPR_64 to SReg_32 copy for an outgoing i1 argument. Adjust
```

Comment:
Typo: SRPR_64

Reply:
This change has been reverted in the latest solution.